src/runtime_src/core/common/config_reader.h

```diff
@@ -355,6 +355,13 @@ get_xma_cpu_mode()
   return value;
 }
 
+inline bool
+get_enable_flat()
+{
+  static unsigned int value = detail::get_bool_value("Runtime.enable_flat",false);
```

Comment:
let's do `static bool`  not that it changes behavior

Reply:
fixed